README.md

```diff
@@ -44,6 +44,35 @@ a token provided by the web app.
 As a starting point please check ```ApiModule``` class - it provides an
 entry point you need to know to communicate with the web application. 
 
+### Troubleshooting
+
+#### Gradle synchronization failure
+
+If you encounter this error:
```

Comment:
There should already be that file named gradle.properties.sample that should just be renamed.